repo/packages/R/registry/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name": "registry",
+  "version": "2.5.1-0.1",
+  "scm": "https://github.com/registry/registry-server.git",
```

Comment:
what exactly doesn't look correct? need 1.7 for persistent storage, registry version is 2.5.1, packaging version is 0.1